### DISCUSSION TITLE
bridge xterio to bsc problem

### DISCUSSION THREAD

**Discussion by l156934030** [blockscout/blockscout#10044] [Support]
https://xterscan.io/tx/0x65f3f3659969550e04851d1e9609730125290ac632caaa86b2651f6bda480b87
It's been 10 days, and I haven't received my money yet.anyone can help me？

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
